pom.xml

```diff
@@ -10,7 +10,7 @@
   </parent>
 
   <artifactId>browserstack-integration</artifactId>
-  <version>1.2.1-SNAPSHOT</version>
+  <version>1.2.2-SNAPSHOT</version>
```

Comment:
This snapshot version change is not required. It will be changed after the release of 1.2.1